packages/features/apps/AdminAppsList.tsx

```diff
@@ -183,26 +185,36 @@ const querySchema = z.object({
 const AdminAppsList = ({
   baseURL,
   className,
+  fromAdmin,
+  currentStep,
+  setIsLoading,
   useQueryParam = false,
 }: {
   baseURL: string;
   className?: string;
+  fromAdmin?: boolean;
+  currentStep?: number;
+  setIsLoading?: Dispatch<SetStateAction<boolean>>;
   useQueryParam?: boolean;
 }) => {
   const router = useRouter();
   return (
     <form
-      id="wizard-step-2"
-      name="wizard-step-2"
+      id={`wizard-step-${currentStep}`}
+      name={`wizard-step-${currentStep}`}
```

Comment:
Why do we need currentStep here other than naming these?

Reply:
Apps list in the wizard can be step 3 or 4 depending on the license you choose, so it needs to be dynamically set.